tabs/osd.js

```diff
@@ -340,10 +342,9 @@ OSD.updateDisplaySize = function() {
 OSD.msp = {
   encodeOther: function() {
     var result = [-1, OSD.data.video_system];
-    if (semver.gte(CONFIG.apiVersion, "1.21.0")) {
+    if (semver.gte(CONFIG.flightControllerVersion, "3.0.1")) {
```

Comment:
@mikeller I also thought we agreed on versions before. Thing is that cleanflight also uses the same api versions, while MSP is different. So the apps still have to do firmware checks